src/core/document.js

```diff
@@ -826,6 +827,53 @@ class PDFDocument {
     return this.xfaFactory !== null;
   }
 
+  loadXfaFonts(handler) {
+    const resources = this.catalog.acroForm.get("DR");
+    const fonts = resources.get("Font");
+    if (!(fonts instanceof Dict)) {
+      return Promise.resolve(null);
+    }
+
+    const objectLoader = new ObjectLoader(resources, ["Font"], this.xref);
+    const partialEvaluator = new PartialEvaluator({
+      xref: this.xref,
+      handler,
+      pageIndex: 0,
```

Comment:
I do wonder if it'd be better to send a proper "dummy" value, such as e.g. `-1`, given that this isn't actually the first page you're parsing here!?